examples/mp-metrics-gauge/README_pt.adoc

```diff
@@ -0,0 +1,153 @@
+= Métricas com MicroProfile `@Gauge` (Medida)
+
+:index-group: MicroProfile
+:jbake-type: page
+:jbake-status: published
+
+Esta é um exemplo sobre como utilizar as métricas de MicroProfile em TomEE.
```

Comment:
The rigth is 'Este é um' instead of 'Esta é um'

Reply:
done thanks